packages/sanity/src/desk/panes/document/statusBar/sparkline/PublishStatus.tsx

```diff
@@ -69,13 +69,8 @@ export function PublishStatus(props: PublishStatusProps) {
           disabled={disabled}
           aria-label={a11yLabel}
           icon={liveEdit ? PlayIcon : PublishIcon}
-          text={
-            collapsed ? null : (
-              <abbr aria-label={a11yLabel}>
-                {liveEdit && lastUpdated ? lastUpdatedTime : lastPublishedTime}
-              </abbr>
-            )
-          }
+          // eslint-disable-next-line no-nested-ternary
+          text={collapsed ? '' : liveEdit && lastUpdated ? lastUpdatedTime : lastPublishedTime}
```

Comment:
Can this be rewritten in a way to avoid a nested ternary?

Reply:
`text={collapsed ? '' : `${liveEdit && lastUpdated ? lastUpdatedTime : lastPublishedTime}`}`
Just joking :smile